### PR TITLE
Addition of Navbar Component

### PR DESCRIPTION
**Pull request by coldn00dles**
# New Pull Request

## SOLVES ISSUE #

- **:pencil: What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Adds a navbar to the website
- **:pencil: What is the current behavior?** (You can also link to an open issue here)
- https://github.com/dokc/rebuilding-explorers-world/issues/5
- **:pencil: What is the new behavior (if this is a feature change)?**
- Visibility changes due to navbar being added
- **:pencil: Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
### :white_check_mark: Check List (Check all the applicable boxes)

- :white_check_mark: I have read [contributing.md](../contributing.md)
- :white_check_mark:I have followed the [code of conduct](../code_of_conduct.md)
- :white_check_mark: My code follows the style guidelines of this project.
- :white_check_mark: I have performed a self-review of my code.
- :white_check_mark: I have commented my code, particularly in hard-to-understand areas.
- :white_check_mark: My changes generate no new warnings.
- :white_check_mark: I have done `git pull` before adding my new changes.
- :white_check_mark: The title of my pull request is a short description of the requested changes.

### :scroll: Screenshots (If Applicable)

Original :
Updated :

### CODE REVIEW

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]
Please make a component for your Navbar and please return them as a component

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]
We are using Astro and not Gatsby, please change accordingly.

**Review by coldn00dles — COMMENTED** [review]
Surely

File: src/components/navbar.astro [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        width : auto;
+      }
+      .centred{
+        justify-content: center;
+        margin:auto;
+        display:block;
+      }
+      .exptext:hover{
+        color:white;
+      }
```
Comment (AbhijithGanesh) [inline src/components/navbar.astro:40]: Can you implement css separately? Or you can use [TailwindCss](https://tailwindcss.com/) to use class based styling.

File: package.json [hunk 2]
```diff
@@ -26,6 +26,5 @@
     "sharp": ">=0.31.0",
     "tailwindcss": "^3.0.24",
     "vue": "^3.2.30"
-  },
-  "devDependencies": {}
```
Comment (AbhijithGanesh) [inline package.json:30]: We donot know why this was deleted, so revert please

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (coldn00dles) [reply package.json:30]: surely

**Review by AbhijithGanesh — APPROVED** [review]
LGTM

**Pull request merged**